tests/Event/ReadModel/JSONLD/EventLDProjectorTest.php

```diff
@@ -1555,6 +1555,54 @@ public function it_keeps_start_date_for_lessen_reeks_on_calendar_updated(): void
         $this->assertEquals($startDate->format(DATE_ATOM), $updatedItem->availableTo);
     }
 
+    /**
+     * @test
+     * @dataProvider typesThatAreAvailableTillStart
+     */
+    public function it_keeps_start_date_for_selected_types_on_calendar_updated(string $termId, string $termName): void
```

Comment:
Is it meaningful to keep `it_keeps_start_date_for_lessen_reeks_on_calendar_updated` still around now that a data provider is used?